specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-04-01/cosmos-db.json

```diff
@@ -1683,6 +1683,86 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/sqlDatabases/{databaseName}/throughputSettings/default/migrateToAutoscale": {
+      "post": {
+        "operationId": "SqlResources_MigrateToAutoscaleSqlDatabase",
+        "x-ms-examples": {
+          "CosmosDBSqlDatabaseMigrateToAutoscale": {
+            "$ref": "./examples/CosmosDBSqlDatabaseMigrateToAutoscale.json"
+          }
+        },
+        "description": "Migrate an Azure Cosmos DB SQL database from manual throughput to autoscale",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/databaseNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The SQL database migrate operation will complete asynchronously."
+          },
+          "200": {
+            "description": "The SQL database migrate operation was completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/ThroughputSettingsGetResults"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/sqlDatabases/{databaseName}/throughputSettings/default/migrateToManualThroughput": {
```

Comment:
>migrateToManualThroughput [](start = 189, length = 25)

If you migrate from from autoscale to manual or vice versa and then do a GET on the throughputSettings resource. Does the response have different properties in each case?

Reply:
Yes. For example, autoscale will return properties such as maxThroughput, etc.